assets/js/modules/analytics-4/components/common/PropertySelect.js

```diff
@@ -82,11 +83,7 @@ export default function PropertySelect( { label } ) {
 		[ propertyID, selectProperty ]
 	);
 
-	if ( ! isValidAccountID( accountID ) ) {
-		return null;
-	}
```

Comment:
Does this really need to be changed? If no account is selected, this component still shouldn't render at all, so this change seems off to me.

Reply:
You are right, this should stay as is. Reverted.